modules/rhcos-setting-grub-password-day-2.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the follow assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="rhcos-setting-grub-password-day-2_{context}"]
+= Setting a GRUB password on {op-system}
+
+:FeatureName: GRUB password management
+include::snippets/technology-preview.adoc[]
+
+You can set up a password to prevent unauthorized users from accessing the GRUB command line, modify kernel command-line arguments, or boot non-default OSTree deployments.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
```

Comment:
Does this functionality work on cluster that were older and upgraded to 4.14? If there are any cornoer case, we may want to document that with keeping in mind node-scale-up happen with older bootimage.

Reply:
We will only support that on clusters installed with 4.14 for now.

Reply:
I can clarify this in the opening paragraph